tests/fast-integration/compass/client.js

```diff
@@ -251,4 +251,19 @@ class DirectorClient {
 module.exports = {
     DirectorConfig,
     DirectorClient
-};
\ No newline at end of file
+};
+
+const retry = async (fn) => {
```

Comment:
Maybe we could name this function `retryOnConcurrentUpdateError` or something like this to add some transparency. What do you think?